### PR TITLE
feat(condo): DOMA-4366 added fullname to billing receipts table

### PR DESCRIPTION
**Pull request by abshnko**
added holder's full name column to the billing receipts table and to the detalization modal

Screenshot 2022-11-09 at 18 36 53
Screenshot 2022-11-09 at 18 22 55

### CODE REVIEW

File: apps/condo/lang/en/en.json [hunk 1]
```diff
@@ -1,1473 +1,1474 @@
...
-  "billing.category.cold_water.name": "Cold water",
-  "billing.category.gas.name": "Gas",
-  "billing.category.trash.name": "Garbage collection",
-  "billing.category.parking.name": "Parking",
-  "billing.category.storage.name": "Storage space",
-  "billing.category.internet.name": "Internet",
-  "billing.category.radio.name": "Radio",
-  "billing.category.intercom.name": "Intercom",
-  "billing.category.telecom.name": "Telecom"
+    "Accept": "Accept",
```
Comment (dkoviazin) [inline apps/condo/lang/en/en.json:2]: You need to change editor settings

Comment (abshnko) [reply apps/condo/lang/en/en.json]: thanks 😅 fixed it

Comment (Alllex202) [reply apps/condo/lang/en/en.json]: you can also squash the commits so they don't contain large diffs. There will be only 1 commit with the necessary changes)

**Review by dkoviazin — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]
Good practice is provide some screenshoots when you had changed something at interface

File: apps/condo/lang/en/en.json (hunk 1 above)
Comment (abshnko) [reply apps/condo/lang/en/en.json]: nice, sounds useful, thanks!

**Comment by abshnko** [comment]
> Good practice is provide some screenshoots when you had changed something at interface

thank you! updated initial comment

File: apps/condo/domains/billing/components/ServicesModal.tsx [hunk 2]
```diff
@@ -113,6 +114,7 @@ export const ServicesModal: React.FC<IServicesModalProps> = ({
             <SubText size={configSize}>
                 {address}{unitName ? `, ${UnitTypePrefix}. ${unitName}` : ''}
             </SubText>
+            <SubText size={configSize}>{fullName}</SubText>
```
Comment (toplenboren) [inline apps/condo/domains/billing/components/ServicesModal.tsx:117]: What if there are no fullname? Will there be a new blank space?

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/billing/components/ServicesModal.tsx (hunk 2 above)
Comment (abshnko) [reply apps/condo/domains/billing/components/ServicesModal.tsx]: good catch, fixed it

**Pull request merged**